2025-05-10 15:46:34,069 | INFO | üìÑ Starting processing file: 2025 - 01 - 01- Night Audit Packet.pdf
2025-05-10 15:46:34,083 | INFO | üìÑ Starting processing file: 2025 - 01 - 02- Night Audit Packet.pdf
2025-05-10 15:46:34,170 | INFO | üìÑ Starting processing file: 2025 - 01 - 03- Night Audit Packet.pdf
2025-05-10 15:46:34,234 | INFO | üìÑ Starting processing file: 2025 - 01 - 04- Night Audit Packet.pdf
2025-05-10 15:46:34,263 | INFO | üìÑ Starting processing file: 2025 - 01 - 05- Night Audit Packet.pdf
2025-05-10 15:46:34,269 | INFO | üìÑ Starting processing file: 2025 - 01 - 06- Night Audit Packet.pdf
2025-05-10 15:46:34,291 | INFO | üìÑ Starting processing file: 2025 - 01 - 07- Night Audit Packet.pdf
2025-05-10 15:46:34,294 | INFO | üìÑ Starting processing file: 2025 - 01 - 08- Night Audit Packet.pdf
2025-05-10 15:46:48,311 | ERROR | ‚ùå A/R Aging extraction failed: insert_dataframe() missing 1 required positional argument: 'filename'
Traceback (most recent call last):
  File "/Users/sandeepmalkanoor/Documents/Python/Night_Audit_DataEngineering_Project/night_audit_etl_pipeline/processor.py", line 64, in handle_section
    insert_dataframe(df, table_name, filename)
TypeError: insert_dataframe() missing 1 required positional argument: 'filename'

2025-05-10 15:46:48,321 | ERROR | ‚ùå Transaction Closeout extraction failed: insert_dataframe() missing 1 required positional argument: 'filename'
Traceback (most recent call last):
  File "/Users/sandeepmalkanoor/Documents/Python/Night_Audit_DataEngineering_Project/night_audit_etl_pipeline/processor.py", line 64, in handle_section
    insert_dataframe(df, table_name, filename)
TypeError: insert_dataframe() missing 1 required positional argument: 'filename'

2025-05-10 15:46:48,328 | ERROR | ‚ùå In-House List extraction failed: insert_dataframe() missing 1 required positional argument: 'filename'
Traceback (most recent call last):
  File "/Users/sandeepmalkanoor/Documents/Python/Night_Audit_DataEngineering_Project/night_audit_etl_pipeline/processor.py", line 64, in handle_section
    insert_dataframe(df, table_name, filename)
TypeError: insert_dataframe() missing 1 required positional argument: 'filename'

2025-05-10 15:46:48,331 | ERROR | ‚ùå room_statistics extraction failed: insert_dataframe() missing 1 required positional argument: 'filename'
Traceback (most recent call last):
  File "/Users/sandeepmalkanoor/Documents/Python/Night_Audit_DataEngineering_Project/night_audit_etl_pipeline/processor.py", line 64, in handle_section
    insert_dataframe(df, table_name, filename)
TypeError: insert_dataframe() missing 1 required positional argument: 'filename'

2025-05-10 15:46:48,332 | ERROR | ‚ùå performance_statistics extraction failed: insert_dataframe() missing 1 required positional argument: 'filename'
Traceback (most recent call last):
  File "/Users/sandeepmalkanoor/Documents/Python/Night_Audit_DataEngineering_Project/night_audit_etl_pipeline/processor.py", line 64, in handle_section
    insert_dataframe(df, table_name, filename)
TypeError: insert_dataframe() missing 1 required positional argument: 'filename'

2025-05-10 15:46:48,333 | ERROR | ‚ùå guest_statistics extraction failed: insert_dataframe() missing 1 required positional argument: 'filename'
Traceback (most recent call last):
  File "/Users/sandeepmalkanoor/Documents/Python/Night_Audit_DataEngineering_Project/night_audit_etl_pipeline/processor.py", line 64, in handle_section
    insert_dataframe(df, table_name, filename)
TypeError: insert_dataframe() missing 1 required positional argument: 'filename'

2025-05-10 15:46:48,338 | ERROR | ‚ùå Ledger Activity extraction failed: insert_dataframe() missing 1 required positional argument: 'filename'
Traceback (most recent call last):
  File "/Users/sandeepmalkanoor/Documents/Python/Night_Audit_DataEngineering_Project/night_audit_etl_pipeline/processor.py", line 64, in handle_section
    insert_dataframe(df, table_name, filename)
TypeError: insert_dataframe() missing 1 required positional argument: 'filename'

2025-05-10 15:46:48,728 | ERROR | ‚ùå A/R Aging extraction failed: insert_dataframe() missing 1 required positional argument: 'filename'
Traceback (most recent call last):
  File "/Users/sandeepmalkanoor/Documents/Python/Night_Audit_DataEngineering_Project/night_audit_etl_pipeline/processor.py", line 64, in handle_section
    insert_dataframe(df, table_name, filename)
TypeError: insert_dataframe() missing 1 required positional argument: 'filename'

2025-05-10 15:46:48,732 | ERROR | ‚ùå Transaction Closeout extraction failed: insert_dataframe() missing 1 required positional argument: 'filename'
Traceback (most recent call last):
  File "/Users/sandeepmalkanoor/Documents/Python/Night_Audit_DataEngineering_Project/night_audit_etl_pipeline/processor.py", line 64, in handle_section
    insert_dataframe(df, table_name, filename)
TypeError: insert_dataframe() missing 1 required positional argument: 'filename'

2025-05-10 15:46:48,735 | ERROR | ‚ùå In-House List extraction failed: insert_dataframe() missing 1 required positional argument: 'filename'
Traceback (most recent call last):
  File "/Users/sandeepmalkanoor/Documents/Python/Night_Audit_DataEngineering_Project/night_audit_etl_pipeline/processor.py", line 64, in handle_section
    insert_dataframe(df, table_name, filename)
TypeError: insert_dataframe() missing 1 required positional argument: 'filename'

2025-05-10 15:46:48,736 | ERROR | ‚ùå room_statistics extraction failed: insert_dataframe() missing 1 required positional argument: 'filename'
Traceback (most recent call last):
  File "/Users/sandeepmalkanoor/Documents/Python/Night_Audit_DataEngineering_Project/night_audit_etl_pipeline/processor.py", line 64, in handle_section
    insert_dataframe(df, table_name, filename)
TypeError: insert_dataframe() missing 1 required positional argument: 'filename'

2025-05-10 15:46:48,737 | ERROR | ‚ùå performance_statistics extraction failed: insert_dataframe() missing 1 required positional argument: 'filename'
Traceback (most recent call last):
  File "/Users/sandeepmalkanoor/Documents/Python/Night_Audit_DataEngineering_Project/night_audit_etl_pipeline/processor.py", line 64, in handle_section
    insert_dataframe(df, table_name, filename)
TypeError: insert_dataframe() missing 1 required positional argument: 'filename'

2025-05-10 15:46:48,747 | ERROR | ‚ùå guest_statistics extraction failed: insert_dataframe() missing 1 required positional argument: 'filename'
Traceback (most recent call last):
  File "/Users/sandeepmalkanoor/Documents/Python/Night_Audit_DataEngineering_Project/night_audit_etl_pipeline/processor.py", line 64, in handle_section
    insert_dataframe(df, table_name, filename)
TypeError: insert_dataframe() missing 1 required positional argument: 'filename'

2025-05-10 15:46:48,750 | ERROR | ‚ùå Ledger Activity extraction failed: insert_dataframe() missing 1 required positional argument: 'filename'
Traceback (most recent call last):
  File "/Users/sandeepmalkanoor/Documents/Python/Night_Audit_DataEngineering_Project/night_audit_etl_pipeline/processor.py", line 64, in handle_section
    insert_dataframe(df, table_name, filename)
TypeError: insert_dataframe() missing 1 required positional argument: 'filename'

2025-05-10 15:46:49,150 | ERROR | ‚ùå A/R Aging extraction failed: insert_dataframe() missing 1 required positional argument: 'filename'
Traceback (most recent call last):
  File "/Users/sandeepmalkanoor/Documents/Python/Night_Audit_DataEngineering_Project/night_audit_etl_pipeline/processor.py", line 64, in handle_section
    insert_dataframe(df, table_name, filename)
TypeError: insert_dataframe() missing 1 required positional argument: 'filename'

2025-05-10 15:46:49,159 | ERROR | ‚ùå Transaction Closeout extraction failed: insert_dataframe() missing 1 required positional argument: 'filename'
Traceback (most recent call last):
  File "/Users/sandeepmalkanoor/Documents/Python/Night_Audit_DataEngineering_Project/night_audit_etl_pipeline/processor.py", line 64, in handle_section
    insert_dataframe(df, table_name, filename)
TypeError: insert_dataframe() missing 1 required positional argument: 'filename'

2025-05-10 15:46:49,162 | ERROR | ‚ùå In-House List extraction failed: insert_dataframe() missing 1 required positional argument: 'filename'
Traceback (most recent call last):
  File "/Users/sandeepmalkanoor/Documents/Python/Night_Audit_DataEngineering_Project/night_audit_etl_pipeline/processor.py", line 64, in handle_section
    insert_dataframe(df, table_name, filename)
TypeError: insert_dataframe() missing 1 required positional argument: 'filename'

2025-05-10 15:46:49,164 | ERROR | ‚ùå room_statistics extraction failed: insert_dataframe() missing 1 required positional argument: 'filename'
Traceback (most recent call last):
  File "/Users/sandeepmalkanoor/Documents/Python/Night_Audit_DataEngineering_Project/night_audit_etl_pipeline/processor.py", line 64, in handle_section
    insert_dataframe(df, table_name, filename)
TypeError: insert_dataframe() missing 1 required positional argument: 'filename'

2025-05-10 15:46:49,165 | ERROR | ‚ùå performance_statistics extraction failed: insert_dataframe() missing 1 required positional argument: 'filename'
Traceback (most recent call last):
  File "/Users/sandeepmalkanoor/Documents/Python/Night_Audit_DataEngineering_Project/night_audit_etl_pipeline/processor.py", line 64, in handle_section
    insert_dataframe(df, table_name, filename)
TypeError: insert_dataframe() missing 1 required positional argument: 'filename'

2025-05-10 15:46:49,166 | ERROR | ‚ùå guest_statistics extraction failed: insert_dataframe() missing 1 required positional argument: 'filename'
Traceback (most recent call last):
  File "/Users/sandeepmalkanoor/Documents/Python/Night_Audit_DataEngineering_Project/night_audit_etl_pipeline/processor.py", line 64, in handle_section
    insert_dataframe(df, table_name, filename)
TypeError: insert_dataframe() missing 1 required positional argument: 'filename'

2025-05-10 15:46:49,171 | ERROR | ‚ùå Ledger Activity extraction failed: insert_dataframe() missing 1 required positional argument: 'filename'
Traceback (most recent call last):
  File "/Users/sandeepmalkanoor/Documents/Python/Night_Audit_DataEngineering_Project/night_audit_etl_pipeline/processor.py", line 64, in handle_section
    insert_dataframe(df, table_name, filename)
TypeError: insert_dataframe() missing 1 required positional argument: 'filename'

2025-05-10 15:46:50,023 | ERROR | ‚ùå A/R Aging extraction failed: insert_dataframe() missing 1 required positional argument: 'filename'
Traceback (most recent call last):
  File "/Users/sandeepmalkanoor/Documents/Python/Night_Audit_DataEngineering_Project/night_audit_etl_pipeline/processor.py", line 64, in handle_section
    insert_dataframe(df, table_name, filename)
TypeError: insert_dataframe() missing 1 required positional argument: 'filename'

2025-05-10 15:46:50,031 | ERROR | ‚ùå Transaction Closeout extraction failed: insert_dataframe() missing 1 required positional argument: 'filename'
Traceback (most recent call last):
  File "/Users/sandeepmalkanoor/Documents/Python/Night_Audit_DataEngineering_Project/night_audit_etl_pipeline/processor.py", line 64, in handle_section
    insert_dataframe(df, table_name, filename)
TypeError: insert_dataframe() missing 1 required positional argument: 'filename'

2025-05-10 15:46:50,041 | ERROR | ‚ùå In-House List extraction failed: insert_dataframe() missing 1 required positional argument: 'filename'
Traceback (most recent call last):
  File "/Users/sandeepmalkanoor/Documents/Python/Night_Audit_DataEngineering_Project/night_audit_etl_pipeline/processor.py", line 64, in handle_section
    insert_dataframe(df, table_name, filename)
TypeError: insert_dataframe() missing 1 required positional argument: 'filename'

2025-05-10 15:46:50,058 | ERROR | ‚ùå room_statistics extraction failed: insert_dataframe() missing 1 required positional argument: 'filename'
Traceback (most recent call last):
  File "/Users/sandeepmalkanoor/Documents/Python/Night_Audit_DataEngineering_Project/night_audit_etl_pipeline/processor.py", line 64, in handle_section
    insert_dataframe(df, table_name, filename)
TypeError: insert_dataframe() missing 1 required positional argument: 'filename'

2025-05-10 15:46:50,061 | ERROR | ‚ùå performance_statistics extraction failed: insert_dataframe() missing 1 required positional argument: 'filename'
Traceback (most recent call last):
  File "/Users/sandeepmalkanoor/Documents/Python/Night_Audit_DataEngineering_Project/night_audit_etl_pipeline/processor.py", line 64, in handle_section
    insert_dataframe(df, table_name, filename)
TypeError: insert_dataframe() missing 1 required positional argument: 'filename'

2025-05-10 15:46:50,064 | ERROR | ‚ùå guest_statistics extraction failed: insert_dataframe() missing 1 required positional argument: 'filename'
Traceback (most recent call last):
  File "/Users/sandeepmalkanoor/Documents/Python/Night_Audit_DataEngineering_Project/night_audit_etl_pipeline/processor.py", line 64, in handle_section
    insert_dataframe(df, table_name, filename)
TypeError: insert_dataframe() missing 1 required positional argument: 'filename'

2025-05-10 15:46:50,068 | ERROR | ‚ùå Ledger Activity extraction failed: insert_dataframe() missing 1 required positional argument: 'filename'
Traceback (most recent call last):
  File "/Users/sandeepmalkanoor/Documents/Python/Night_Audit_DataEngineering_Project/night_audit_etl_pipeline/processor.py", line 64, in handle_section
    insert_dataframe(df, table_name, filename)
TypeError: insert_dataframe() missing 1 required positional argument: 'filename'

2025-05-10 15:46:50,575 | ERROR | ‚ùå A/R Aging extraction failed: insert_dataframe() missing 1 required positional argument: 'filename'
Traceback (most recent call last):
  File "/Users/sandeepmalkanoor/Documents/Python/Night_Audit_DataEngineering_Project/night_audit_etl_pipeline/processor.py", line 64, in handle_section
    insert_dataframe(df, table_name, filename)
TypeError: insert_dataframe() missing 1 required positional argument: 'filename'

2025-05-10 15:46:50,600 | ERROR | ‚ùå Transaction Closeout extraction failed: insert_dataframe() missing 1 required positional argument: 'filename'
Traceback (most recent call last):
  File "/Users/sandeepmalkanoor/Documents/Python/Night_Audit_DataEngineering_Project/night_audit_etl_pipeline/processor.py", line 64, in handle_section
    insert_dataframe(df, table_name, filename)
TypeError: insert_dataframe() missing 1 required positional argument: 'filename'

2025-05-10 15:46:50,626 | ERROR | ‚ùå In-House List extraction failed: insert_dataframe() missing 1 required positional argument: 'filename'
Traceback (most recent call last):
  File "/Users/sandeepmalkanoor/Documents/Python/Night_Audit_DataEngineering_Project/night_audit_etl_pipeline/processor.py", line 64, in handle_section
    insert_dataframe(df, table_name, filename)
TypeError: insert_dataframe() missing 1 required positional argument: 'filename'

2025-05-10 15:46:50,628 | ERROR | ‚ùå room_statistics extraction failed: insert_dataframe() missing 1 required positional argument: 'filename'
Traceback (most recent call last):
  File "/Users/sandeepmalkanoor/Documents/Python/Night_Audit_DataEngineering_Project/night_audit_etl_pipeline/processor.py", line 64, in handle_section
    insert_dataframe(df, table_name, filename)
TypeError: insert_dataframe() missing 1 required positional argument: 'filename'

2025-05-10 15:46:50,629 | ERROR | ‚ùå performance_statistics extraction failed: insert_dataframe() missing 1 required positional argument: 'filename'
Traceback (most recent call last):
  File "/Users/sandeepmalkanoor/Documents/Python/Night_Audit_DataEngineering_Project/night_audit_etl_pipeline/processor.py", line 64, in handle_section
    insert_dataframe(df, table_name, filename)
TypeError: insert_dataframe() missing 1 required positional argument: 'filename'

2025-05-10 15:46:50,630 | ERROR | ‚ùå guest_statistics extraction failed: insert_dataframe() missing 1 required positional argument: 'filename'
Traceback (most recent call last):
  File "/Users/sandeepmalkanoor/Documents/Python/Night_Audit_DataEngineering_Project/night_audit_etl_pipeline/processor.py", line 64, in handle_section
    insert_dataframe(df, table_name, filename)
TypeError: insert_dataframe() missing 1 required positional argument: 'filename'

2025-05-10 15:46:50,633 | ERROR | ‚ùå Ledger Activity extraction failed: insert_dataframe() missing 1 required positional argument: 'filename'
Traceback (most recent call last):
  File "/Users/sandeepmalkanoor/Documents/Python/Night_Audit_DataEngineering_Project/night_audit_etl_pipeline/processor.py", line 64, in handle_section
    insert_dataframe(df, table_name, filename)
TypeError: insert_dataframe() missing 1 required positional argument: 'filename'

2025-05-10 15:46:51,456 | ERROR | ‚ùå A/R Aging extraction failed: insert_dataframe() missing 1 required positional argument: 'filename'
Traceback (most recent call last):
  File "/Users/sandeepmalkanoor/Documents/Python/Night_Audit_DataEngineering_Project/night_audit_etl_pipeline/processor.py", line 64, in handle_section
    insert_dataframe(df, table_name, filename)
TypeError: insert_dataframe() missing 1 required positional argument: 'filename'

2025-05-10 15:46:51,461 | ERROR | ‚ùå Transaction Closeout extraction failed: insert_dataframe() missing 1 required positional argument: 'filename'
Traceback (most recent call last):
  File "/Users/sandeepmalkanoor/Documents/Python/Night_Audit_DataEngineering_Project/night_audit_etl_pipeline/processor.py", line 64, in handle_section
    insert_dataframe(df, table_name, filename)
TypeError: insert_dataframe() missing 1 required positional argument: 'filename'

2025-05-10 15:46:51,472 | ERROR | ‚ùå In-House List extraction failed: insert_dataframe() missing 1 required positional argument: 'filename'
Traceback (most recent call last):
  File "/Users/sandeepmalkanoor/Documents/Python/Night_Audit_DataEngineering_Project/night_audit_etl_pipeline/processor.py", line 64, in handle_section
    insert_dataframe(df, table_name, filename)
TypeError: insert_dataframe() missing 1 required positional argument: 'filename'

2025-05-10 15:46:51,492 | ERROR | ‚ùå room_statistics extraction failed: insert_dataframe() missing 1 required positional argument: 'filename'
Traceback (most recent call last):
  File "/Users/sandeepmalkanoor/Documents/Python/Night_Audit_DataEngineering_Project/night_audit_etl_pipeline/processor.py", line 64, in handle_section
    insert_dataframe(df, table_name, filename)
TypeError: insert_dataframe() missing 1 required positional argument: 'filename'

2025-05-10 15:46:51,499 | ERROR | ‚ùå performance_statistics extraction failed: insert_dataframe() missing 1 required positional argument: 'filename'
Traceback (most recent call last):
  File "/Users/sandeepmalkanoor/Documents/Python/Night_Audit_DataEngineering_Project/night_audit_etl_pipeline/processor.py", line 64, in handle_section
    insert_dataframe(df, table_name, filename)
TypeError: insert_dataframe() missing 1 required positional argument: 'filename'

2025-05-10 15:46:51,501 | ERROR | ‚ùå guest_statistics extraction failed: insert_dataframe() missing 1 required positional argument: 'filename'
Traceback (most recent call last):
  File "/Users/sandeepmalkanoor/Documents/Python/Night_Audit_DataEngineering_Project/night_audit_etl_pipeline/processor.py", line 64, in handle_section
    insert_dataframe(df, table_name, filename)
TypeError: insert_dataframe() missing 1 required positional argument: 'filename'

2025-05-10 15:46:51,503 | ERROR | ‚ùå Ledger Activity extraction failed: insert_dataframe() missing 1 required positional argument: 'filename'
Traceback (most recent call last):
  File "/Users/sandeepmalkanoor/Documents/Python/Night_Audit_DataEngineering_Project/night_audit_etl_pipeline/processor.py", line 64, in handle_section
    insert_dataframe(df, table_name, filename)
TypeError: insert_dataframe() missing 1 required positional argument: 'filename'

2025-05-10 15:46:51,665 | ERROR | ‚ùå A/R Aging extraction failed: insert_dataframe() missing 1 required positional argument: 'filename'
Traceback (most recent call last):
  File "/Users/sandeepmalkanoor/Documents/Python/Night_Audit_DataEngineering_Project/night_audit_etl_pipeline/processor.py", line 64, in handle_section
    insert_dataframe(df, table_name, filename)
TypeError: insert_dataframe() missing 1 required positional argument: 'filename'

2025-05-10 15:46:51,676 | ERROR | ‚ùå Transaction Closeout extraction failed: insert_dataframe() missing 1 required positional argument: 'filename'
Traceback (most recent call last):
  File "/Users/sandeepmalkanoor/Documents/Python/Night_Audit_DataEngineering_Project/night_audit_etl_pipeline/processor.py", line 64, in handle_section
    insert_dataframe(df, table_name, filename)
TypeError: insert_dataframe() missing 1 required positional argument: 'filename'

2025-05-10 15:46:51,679 | ERROR | ‚ùå In-House List extraction failed: insert_dataframe() missing 1 required positional argument: 'filename'
Traceback (most recent call last):
  File "/Users/sandeepmalkanoor/Documents/Python/Night_Audit_DataEngineering_Project/night_audit_etl_pipeline/processor.py", line 64, in handle_section
    insert_dataframe(df, table_name, filename)
TypeError: insert_dataframe() missing 1 required positional argument: 'filename'

2025-05-10 15:46:51,681 | ERROR | ‚ùå room_statistics extraction failed: insert_dataframe() missing 1 required positional argument: 'filename'
Traceback (most recent call last):
  File "/Users/sandeepmalkanoor/Documents/Python/Night_Audit_DataEngineering_Project/night_audit_etl_pipeline/processor.py", line 64, in handle_section
    insert_dataframe(df, table_name, filename)
TypeError: insert_dataframe() missing 1 required positional argument: 'filename'

2025-05-10 15:46:51,683 | ERROR | ‚ùå performance_statistics extraction failed: insert_dataframe() missing 1 required positional argument: 'filename'
Traceback (most recent call last):
  File "/Users/sandeepmalkanoor/Documents/Python/Night_Audit_DataEngineering_Project/night_audit_etl_pipeline/processor.py", line 64, in handle_section
    insert_dataframe(df, table_name, filename)
TypeError: insert_dataframe() missing 1 required positional argument: 'filename'

2025-05-10 15:46:51,683 | ERROR | ‚ùå guest_statistics extraction failed: insert_dataframe() missing 1 required positional argument: 'filename'
Traceback (most recent call last):
  File "/Users/sandeepmalkanoor/Documents/Python/Night_Audit_DataEngineering_Project/night_audit_etl_pipeline/processor.py", line 64, in handle_section
    insert_dataframe(df, table_name, filename)
TypeError: insert_dataframe() missing 1 required positional argument: 'filename'

2025-05-10 15:46:51,686 | ERROR | ‚ùå Ledger Activity extraction failed: insert_dataframe() missing 1 required positional argument: 'filename'
Traceback (most recent call last):
  File "/Users/sandeepmalkanoor/Documents/Python/Night_Audit_DataEngineering_Project/night_audit_etl_pipeline/processor.py", line 64, in handle_section
    insert_dataframe(df, table_name, filename)
TypeError: insert_dataframe() missing 1 required positional argument: 'filename'

